indico/modules/rb_new/util.py

```diff
@@ -157,11 +162,10 @@ def get_existing_room_occurrences(room, start_dt, end_dt, allow_overlapping=Fals
     return query.all()
 
 
-def get_room_conflicts(room, start_dt, end_dt, repeat_frequency, repeat_interval):
+def get_room_conflicts(room, candidates, blockings, nonbookable_periods, unbookable_hours):
```

Comment:
This function has changed a bit since my recent commits, so I guess you will have to deal with a few conflicts